source/includes/_users.md

```diff
@@ -63,6 +66,9 @@ Attribute | Description
 `first_name` <br> *string*      | The user's first name
 `last_name` <br> *string*       | The user's last name
 `email` <br> *string*           | The user's e-mail address
+`account_type` <br> *string*    | The user's account type.<br> Possible values are `global`, `linked` and `local`
+`display_name` <br> *string*    | The user's display name<br>simplified version of the `username`
+`auth_provider_types` <br> *array of string*    | The list of potential authentification providers type for this user.<br> Currently `saml` is the only available provider.
```

Comment:
I wouldn't say `potential`, even though I understand you mean that they could authenticate with either, but only if they choose so.

Reply:
As the list is not really a complete list of the user options (we never mention `central` in the list), I guess I'll just drop the `potential` part.